README.md

```diff
@@ -1,2 +1,10 @@
 # frinder
 Website for finding friends based on your common interests
+
```

Comment:
Would be great to have node version specified both in the readme and package.json (engine section)

Reply:
done